src/store/trails/mutations.js

```diff
@@ -8,10 +8,9 @@ export const setFees = (state, config) => {
 };
 
 export const resetBallots = (state) => {
-  console.log(`resetballots called`);
-  state.ballots.list.data = [];
-  state.ballots.list.loaded = false;
-  console.log(`ballots reset`);
+  // THere's no need to reset what's already fetched. Just use filters.
+  // state.ballots.list.data = [];
+  // state.ballots.list.loaded = false;
 };
```

Comment:
Is this _temporarily_ commented out? If not, we can remove unused method.

Reply:
Please add to new issue as it does not seem to be a blocker for relaunch.